.rubocop.yml

```diff
@@ -95,4 +95,4 @@ Style/AsciiComments:
 Style/IfUnlessModifier:
   Enabled: false
 Style/SymbolProc:
-  Enabled: false
+  Enabled: false
```

Comment:
Can you remove the line break at the end of this line.

Reply:
Thanks Paula, it looks like it was actually missing a line break, which was why github was highlighting it with an icon.

I've added a line break, should look ok now :)

Reply:
It now looks to be consistent with the previous .rubocop.yml hat TP had in, that we were using as reference, so won't show on the diff :)